zh/backup-to-aws-s3-using-br.md

```diff
@@ -318,7 +318,7 @@ Amazon S3 支持以下几种 `storageClass` 类型：
 更多 `Backup` CR 字段的详细解释:
 
 * `.spec.metadata.namespace`：`Backup` CR 所在的 namespace。
-* `.spec.from.host`：待备份 TiDB 集群的访问地址。
+* `.spec.from.host`：待备份 TiDB 集群的访问地址，为需要导出的 TiDB 的 service name，例如 `basic-tidb`。
```

Comment:
Please also help update `zh/backup-to-gcs.md` and `zh/backup-to-gcs-using-br.md`.

Reply:
addressed in https://github.com/pingcap/docs-tidb-operator/pull/523/commits/1f2a84c411eafd64139ce6b729e8bc7f10e974dc